connection/TypeDBClientImpl.java

```diff
@@ -42,15 +42,11 @@
 
     private static final String TYPEDB_CLIENT_RPC_THREAD_NAME = "typedb-client-rpc";
 
-    private final ManagedChannel channel;
```

Comment:
Move the definition of `channel` and `stub` into the respective concrete classes: `CoreClient` and `ClusterServer` client